### PR TITLE
Improved the test coverage through updates to the unit tests and log …

### PR DESCRIPTION
**Pull request by dtoulcher-ch**
…messages within methods

Through improvements and utilising the PiTest plugin:
 * Line coverage has been improved by 7%      (98%)
 * Test Strength has been improved by 8%      (86%)
 * Mutations killed has been improved by 14%. (83%)

CC-1077

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/authorization/UserAuthorizationInterceptor.java [hunk 1]
```diff
@@ -34,12 +35,12 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
 
         if(createdByUserEmail.equals(requestUserEmail)) {
             apiLogger.debugContext(requestId, "User is authorised to access objection");
-            return true;
+            userIsAuthorised = true;
+        }else {
```
Comment (bcullerton) [inline src/main/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/authorization/UserAuthorizationInterceptor.java:39]: Spacing here

File: src/test/java/uk/gov/companieshouse/api/strikeoffobjections/client/OracleQueryClientTest.java [hunk 2]
```diff
@@ -17,8 +17,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (bcullerton) [inline src/test/java/uk/gov/companieshouse/api/strikeoffobjections/client/OracleQueryClientTest.java:20]: .* import

File: src/test/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/authorization/UserAuthorizationInterceptorTest.java [hunk 3]
```diff
@@ -4,21 +4,21 @@
...
+import uk.gov.companieshouse.api.strikeoffobjections.service.impl.ERICHeaderFields;
 import uk.gov.companieshouse.api.strikeoffobjections.service.impl.ERICHeaderParser;
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (bcullerton) [inline src/test/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/authorization/UserAuthorizationInterceptorTest.java:21]: .*

**Review by bcullerton — COMMENTED** [review]
Generally looks good, just minor formatting comments

**Review by bcullerton — APPROVED** [review]

**Pull request merged**